recipes/libxml2/all/conanfile.py

```diff
@@ -255,9 +257,13 @@ def package_info(self):
             self.env_info.PATH.append(bindir)
 
         if self.settings.os == "Linux" or self.settings.os == "Macos":
-            self.cpp_info.system_libs.append('m')
```

Comment:
Is it really required for Macos?

Reply:
I don't know of another recipe that adds `m` for Macos.
I can't check it...

Reply:
that's good catch, it shouldn't be needed on macOS - everything reside in libSystem.dylib